File: src/java/main/io/github/_0xmaxlab/commands/RichJson_file.java
package io.github._0xmaxlab.commands;

import io.github._0xmaxlab.core.RichJsonCommand;
import io.github._0xmaxlab.core.RichJsonContext;
import io.github._0xmaxlab.helper.RichJsonFileHelper;
import io.github._0xmaxlab.core.RichJsonParser;

public class RichJson_file implements RichJsonCommand {
    @Override
    public Object execute(RichJsonParser parser, RichJsonContext context) {
        String fileName = context.currentMember + ".json";
        return RichJsonFileHelper.readFile(fileName, true);
    }
}

File: src/c++/main/commands/RichJson_file.cpp
#include "RichJson_file.hpp"
#include "../core/RichJsonContext.hpp"
#include "../helper/RichJsonFileHelper.hpp"

namespace RichJson {

json richJson_file(RichJsonParser&, RichJsonContext& context) {
    std::string fileName = context.currentMember.get<std::string>() + ".json";
    return RichJsonFileHelper::readFile(fileName, true);
}

}
